src/main/java/net/glowstone/block/blocktype/BlockLiquid.java

```diff
@@ -255,4 +254,13 @@ public boolean canTickRandomly() {
         return true;
     }
 
+    @Override
+    public boolean isPulseOnce(GlowBlock block) {
+        return false;
```

Comment:
Water is a pulse once block, since each flow from one block to the other only happens once.